src/frontend/komponenter/Fagsak/Søknad/BarnMedOpplysninger.tsx

```diff
@@ -22,13 +33,35 @@ const BarnMedOpplysninger: React.FunctionComponent<IProps> = ({ barn }) => {
         );
 
     return (
-        <div className={'søknad__barna__barn-rad'}>
+        <BarnMedOpplysningerWrapper>
             <FamilieCheckbox
                 erLesevisning={lesevisning}
                 id={`barn-${barn.ident}`}
-                label={`${barn.navn ?? 'Navn ukjent'} (${hentAlderSomString(
-                    barn.fødselsdato
-                )}) | ${formaterPersonIdent(barn.ident)}`}
+                label={
```

Comment:
Det var hacket litt til med styling for å få checkbox label og fjernknappen til å være på samme linje. Så nå puttet jeg fjernknappen sammen med labelen. Har ikke gjort dette før, men tror det skal gå bra? eller er det enda mer hacky?

Reply:
Dette er supert tenker jeg 👍